generic/stressapptest.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Harish <harish@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive, build, cpu, memory, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class StressAppTest(Test):
+
+    def setUp(self):
+        self.test_file = self.params.get('tmp_file', default='/tmp/dummy')
+        self.duration = self.params.get('duration', default='30')
+        self.threads = self.params.get(
+            'threads', default=cpu.online_cpus_count())
+        self.size = self.params.get(
+            'memory_to_test', default=int(0.9 * memory.meminfo.MemFree.m))
+
+        smm = SoftwareManager()
+        deps = ['gcc', 'libtool', 'autoconf', 'automake', 'make']
+        for package in deps:
```

Comment:
can be like 
`for package ['gcc', 'libtool', 'autoconf', 'automake', 'make']` to avoid extra variable to  minimize garbage collector job